### PR TITLE
WB-1134.3: Add stories for MultiSelect

### PR DESCRIPTION
**Pull request by jandrade**
## Summary:

Last part of the wb-dropdown migration to storybook:

- Adding stories for `MultiSelect`.
- Adding props table for `MultiSelect`, `OptionItem` and `SeparatorItem`.

Issue: WB-1134

## Test plan:

Open storybook and navigate to Dropdown > MultiSelect.

Verify that all the examples work.
Screen Shot 2022-06-01 at 6 13 39 PM
Screen Shot 2022-06-01 at 6 13 51 PM

### CODE REVIEW

**Comment by jandrade** [comment]
The parent pull-request ([#1512](https://github.com/Khan/wonder-blocks/pull/1512)) has been merged into `main`, but this branch (`sb-dd2`) now has conflicts with the new base branch. These conflicts must be resolved before checks can complete on this pull-request.

File: packages/wonder-blocks-dropdown/src/components/__docs__/multi-select.stories.js [hunk 1]
```diff
@@ -7,10 +7,195 @@ import {View} from "@khanacademy/wonder-blocks-core";
...
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+import multiSelectArgtypes from "./base-select.argtypes.js";
+import {defaultLabels} from "../../util/constants.js";
 
 export default {
     title: "Dropdown / MultiSelect",
+    component: MultiSelect,
+    argTypes: {
+        ...multiSelectArgtypes,
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/src/components/__docs__/multi-select.stories.js:27]: Cool!

File: packages/wonder-blocks-dropdown/src/components/__docs__/multi-select.stories.js [hunk 2]
```diff
@@ -7,10 +7,195 @@ import {View} from "@khanacademy/wonder-blocks-core";
...
+    args: {
+        isFilterable: false,
+        opened: false,
+        disabled: false,
+        light: false,
+        shortcuts: false,
+        implicitAllEnabled: false,
+        id: "",
+        testId: "",
+    },
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/src/components/__docs__/multi-select.stories.js:41]: These are the default props for the args listed in `argTypes`?

File: packages/wonder-blocks-dropdown/src/components/__docs__/multi-select.stories.js [hunk 3]
```diff
@@ -7,10 +7,195 @@ import {View} from "@khanacademy/wonder-blocks-core";
...
+    decorators: [
+        (Story: StoryComponentType): React.Element<typeof View> => (
+            <View style={styles.example}>
+                <Story />
+            </View>
+        ),
```
Comment (kevinbarabash) [inline packages/wonder-blocks-dropdown/src/components/__docs__/multi-select.stories.js:47]: Does this behave differently that adding the `<View style={styles.example}>` as a wrapper in the `Template` below?

**Review by kevinbarabash — APPROVED** [review]
LGTM.  Thanks for adding stories for MultiSelect. 🎉 

Is there anyway to view the Storybook preview without checking out the branch?

**Pull request merged**